.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: php
+
+dist: trusty
+
+env:
+  global:
+    - COMPOSER_ROOT_VERSION=3.x-dev
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 7.1.2
+      env: DB=PGSQL PHPUNIT_TEST=1 PHPCS_TEST=1
+    - php: 7.1.2
+      env: DB=MYSQL PHPUNIT_TEST=1
+    - php: 7.1.2
+      env: DB=MYSQL PDO=1 PHPUNIT_COVERAGE_TEST=1
```

Comment:
Would be cool to add various core recipe versions to this - see supported modules for example